combinations/hash.tsv

```diff
@@ -43,5 +43,5 @@ array-as-vcf=1.0.0,tabix=0.2.6
 array-as-vcf=1.0.1,tabix=0.2.6
 cyvcf2=0.11.5,click=6.7
 vtools=1.0.0,jq=1.6
-pgx-variant-tools=0.0.2,locus_processing=0.0.4,snakemake-minimal=5.10.0,biopython=1.76,pyyaml=3.12
+pgx-variant-tools=0.0.2-py_2,locus_processing=0.0.4,snakemake-minimal=5.10.0,biopython=1.76,pyyaml=3.12
```

Comment:
```suggestion
pgx-variant-tools=0.0.2=py_2,locus_processing=0.0.4,snakemake-minimal=5.10.0,biopython=1.76,pyyaml=3.12
```
but we'd lose the build number. Let me add the missing column, then we can do this.

Reply:
Thanks for looking into this. Should I wait with the commit or not?

Reply:
You can commit, but I guess we will not see a new container published. I'm working on it but it might take a little while.